pkg/machines/libvirt-common.js

```diff
@@ -146,6 +146,18 @@ export function getDiskElemByTarget(domxml, targetOriginal) {
     }
 }
 
+function getNetworkElem(netXml) {
```

Comment:
The functions in this file are sorted according to their name. It's nitpicking, I know, but if you don't mind to move them to correct line it would make me happier :)